src/display/api.js

```diff
@@ -299,6 +302,9 @@ PDFJS.getDocument = function getDocument(src,
     source = src;
   }
 
+  // Default range chunk size to 2^16 = 65536
+  source.rangeChunkSize = source.rangeChunkSize || 65536;
```

Comment:
Done, decided against adding the default in the ChunkedStreamManager constructor since we still need that value in getPdfManager() in worker.js.